client/src/components/InputsModal/AuthSettings.ts

```diff
@@ -249,7 +249,10 @@ export const getAccessFields = (
       title: 'Client ID',
       description: 'Client ID provided during registration of Inferno',
       optional: true,
-      hide: !accessValues.get('refresh_token'),
+      hide:
+        authType === 'backend_services'
```

Comment:
Could you use this logic to just assign `'access_token'` or `'refresh_token'` to a variable that gets used here rather than repeating the logic for each field?

Reply:
Yep totally works -- updated.